pstress/mysqld_options_ms_80.txt

```diff
@@ -2958,8 +3469,101 @@
+--replica-checkpoint-group=1125899906842624
+--replica-checkpoint-period=300
+--replica-checkpoint-period=0
+--replica-checkpoint-period=1
+--replica-checkpoint-period=2
+--replica-checkpoint-period=12
+--replica-checkpoint-period=24
+--replica-checkpoint-period=254
+--replica-checkpoint-period=1023
+--replica-checkpoint-period=2047
+--replica-checkpoint-period=-1125899906842624
+--replica-checkpoint-period=1125899906842624
+--replica-compressed-protocol=1
+--replica-compressed-protocol=0
+--replica-exec-mode=STRICT
+--replica-exec-mode=IDEMPOTENT
+--replica-load-tmpdir=tmp
+--replica-max-allowed-packet=1073741824
+--replica-max-allowed-packet=0
+--replica-max-allowed-packet=1
+--replica-max-allowed-packet=2
+--replica-max-allowed-packet=12
+--replica-max-allowed-packet=24
+--replica-max-allowed-packet=254
+--replica-max-allowed-packet=1023
+--replica-max-allowed-packet=2047
+--replica-max-allowed-packet=-1125899906842624
+--replica-max-allowed-packet=1125899906842624
+--replica-net-timeout=60
+--replica-net-timeout=0
+--replica-net-timeout=1
+--replica-net-timeout=2
+--replica-net-timeout=12
+--replica-net-timeout=24
+--replica-net-timeout=254
+--replica-net-timeout=1023
+--replica-net-timeout=2047
+--replica-net-timeout=-1125899906842624
+--replica-net-timeout=1125899906842624
+--replica-parallel-type=DATABASE
+--replica-parallel-type=LOGICAL_CLOCK
+--replica-parallel-workers=0
+--replica-parallel-workers=1
+--replica-parallel-workers=2
+--replica-parallel-workers=12
+--replica-parallel-workers=24
+--replica-parallel-workers=254
+--replica-parallel-workers=1023
+--replica-parallel-workers=2047
+--replica-parallel-workers=-1125899906842624
+--replica-parallel-workers=1125899906842624
+--replica-pending-jobs-size-max=134217728
+--replica-pending-jobs-size-max=0
+--replica-pending-jobs-size-max=1
+--replica-pending-jobs-size-max=2
+--replica-pending-jobs-size-max=12
+--replica-pending-jobs-size-max=24
+--replica-pending-jobs-size-max=254
+--replica-pending-jobs-size-max=1023
+--replica-pending-jobs-size-max=2047
+--replica-pending-jobs-size-max=-1125899906842624
+--replica-pending-jobs-size-max=1125899906842624
+--replica-preserve-commit-order=1
+--replica-preserve-commit-order=0
+--replica-sql-verify-checksum=1
+--replica-sql-verify-checksum=0
+--replica-transaction-retries=10
+--replica-transaction-retries=0
+--replica-transaction-retries=1
+--replica-transaction-retries=2
+--replica-transaction-retries=12
+--replica-transaction-retries=24
+--replica-transaction-retries=254
+--replica-transaction-retries=1023
+--replica-transaction-retries=2047
+--replica-transaction-retries=-1125899906842624
+--replica-transaction-retries=1125899906842624
 --replicate-same-server-id=1
 --replicate-same-server-id=0
+--replication-optimize-for-static-plugin-config=1
+--replication-optimize-for-static-plugin-config=0
+--replication-sender-observe-commit-only=1
+--replication-sender-observe-commit-only=0
 --report-port=0
```

Comment:
Must be excluded

Reply:
Hello Mohit , 
All reviews addressed , kindly check and let me know if anything needs to be addressed.

Reply:
You forgot to exclude --report-port.